shared/pkg/utils/input.go

```diff
@@ -41,9 +41,16 @@ func ValidateInputType(inputType string) (SourceType, error) {
 	}
 }
 
-func CreateUserInput(sourceType SourceType, src string) string {
+func CreateSource(sourceType SourceType, src string, localImage bool) string {
 	if sourceType != IMAGE {
 		src = fmt.Sprintf("%s:%s", sourceType, src)
```

Comment:
We should return here and not continue to `setImageSource`. (or use a switch case on `sourceType`)
Better to add a unit test here.

Reply:
oops, thx , I will add unit test